src/screens/SUE/SueListScreen.tsx

```diff
@@ -81,7 +121,25 @@ export const SueListScreen = ({ navigation, route }: Props) => {
   const listItems = useMemo(() => {
     if (!data?.length) return [];
 
-    return parseListItemsFromQuery(query, data, undefined, { appDesignSystem }).reverse();
+    let parsedListItem = parseListItemsFromQuery(query, data, undefined, {
+      appDesignSystem
+    });
+
+    if (queryVariables.sortBy) {
+      const { sortBy } = queryVariables;
+
+      if (sortBy === SORT_BY.REQUESTED_DATE_TIME || sortBy === SORT_BY.UPDATED_DATE_TIME) {
+        parsedListItem = _sortBy(parsedListItem, (item) => new Date(item[sortBy]));
+      } else {
+        parsedListItem = _sortBy(parsedListItem, sortBy);
+      }
+    }
+
+    if (!queryVariables.sortBy || queryVariables.sortBy !== SORT_BY.TITLE) {
+      parsedListItem = _reverse(parsedListItem);
```

Comment:
what is the benefit of using lodashs reverse over the native reverse?

Reply:
Actually, I don't think there is any advantage, but since we use `lodash` in `SueListScreen`, I wanted to use reverse with `lodash`.